src/o365/spo/commands/page/page-get.ts

```diff
@@ -65,18 +63,32 @@ class SpoPageGetCommand extends SpoCommand {
           });
         });
 
-        const page: any = {
-          commentsDisabled: res.ListItemAllFields.CommentsDisabled,
-          numSections: clientSidePage.sections.length,
-          numControls: numControls,
-          title: res.ListItemAllFields.Title
-        };
+        if (args.options.output === 'json') {
```

Comment:
To avoid duplicating the code, I changed the `if` block to:

```ts
let page: any = {
  commentsDisabled: res.ListItemAllFields.CommentsDisabled,
  numSections: clientSidePage.sections.length,
  numControls: numControls,
  title: res.ListItemAllFields.Title
};

if (res.ListItemAllFields.PageLayoutType) {
  page.layoutType = res.ListItemAllFields.PageLayoutType;
}

if (args.options.output === 'json') {
  page = Object.assign(res, page);
}
  
cmd.log(page);
```